js/apps/system/_admin/aardvark/APP/react/src/views/g6graphs/Headerinfo.js

```diff
@@ -0,0 +1,253 @@
+                createEl.href = canvasUrl;
+                createEl.download = `${graphName}`;
+                createEl.click();
+                createEl.remove();
+              }}
+              style={{
+                'background': '#fff',
+                'border': 0,
+                'marginLeft': 'auto'
+              }}><i class="fa fa-download" style={{ 'fontSize': '18px', 'marginTop': '6px', 'color': '#555' }}></i>
+            </button>
+          </ToolTip>
+
+          <ToolTip
+            title={"Enter full screen"}
+            setArrow={true}
+          >
+            <button
+              onClick={() => {
+                const elem = document.getElementById("visnetworkdiv");
+                enterFullscreen(elem);
+              }}
+              style={{
+                'background': '#fff',
+                'border': 0
+              }}><i class="fa fa-arrows-alt" style={{ 'fontSize': '18px', 'marginTop': '6px', 'color': '#555' }}></i>
+            </button>
+          </ToolTip>
+
+          <ToolTip
+            title={"Switch to the old graph viewer"}
+            setArrow={true}
+          >
+            <button
+              onClick={() => {
+                window.location.href = `/_db/_system/_admin/aardvark/index.html#graph/${graphName}`;
+              }}
+              style={{
+                'background': '#fff',
+                'border': 0
+              }}><i class="fa fa-retweet" style={{ 'fontSize': '18px', 'marginTop': '6px', 'color': '#555' }}></i>
+            </button>
+          </ToolTip>
+
+          <ToolTip
+            title={"Get instructions and support on how to use the graph viewer"}
+            setArrow={true}
+          >
+            <button
+              onClick={() => {
+                setShowHelpModal(true);
+              }}
+              style={{
+                'background': '#fff',
+                'border': 0
+              }}><i class="fa fa-question-circle" style={{ 'fontSize': '18px', 'marginTop': '6px', 'color': '#555' }}></i>
+            </button>
+          </ToolTip>
+        
+        <IconButton icon={'bars'} onClick={() => {
+          toggleDrawer(!open)
+        }}
+          style={{
+          background: '#2ECC71',
+          color: 'white',
+          paddingLeft: '14px',
+          marginLeft: 'auto'
+        }}>
+          Settings
+        </IconButton>
+        </div>
+      </div>
+      <Drawer
+        position="right"
+        open={open}
+       >
+        <div style={{ 'background': '#404a53' }}>
+          <div style={{ 'padding': '24px', 'display': 'flex' }}>
+            <ButtonSave
+              graphName={graphName}
+              onGraphDataLoaded={(newGraphData, responseTimesObject) => {
+                onGraphDataLoaded(newGraphData, responseTimesObject)}
+              }
+              onIsLoadingData={(isLoadingData) => setIsLoadingData(isLoadingData)}
+            />
+          </div>
+          <AccordionView
```

Comment:
Let's switch to using the Chakra `<Accordion>`. See `views/views/ViewLeftPane` for a sample usage. We can get rid of all these custom components after that (everything added under `components/Accordion`)